src/main/java/org/prebid/server/bidder/conversant/ConversantBidder.java

```diff
@@ -175,7 +142,9 @@ private static Imp modifyImp(Imp imp, ExtImpConversant impExt) {
         final Imp.ImpBuilder impBuilder = imp.toBuilder();
```

Comment:
Pls place the builder creation inside `if` condition where we can do return.

Reply:
Actually, there is no sense of  'if creation', because of this lines `.displaymanager(DISPLAY_MANAGER)
                .displaymanagerver(DISPLAY_MANAGER_VER)`(Builder will always be created)